invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/util.js

```diff
@@ -15,14 +15,22 @@ import _camelCase from "lodash/camelCase";
 export function createSearchAppInit(
   defaultComponents,
   autoInit = true,
-  autoInitDataAttr = "invenio-search-config"
+  autoInitDataAttr = "invenio-search-config",
```

Comment:
Not related to this PR: can you please improve the code decoupling the id/attribute of the `rootElement` and the attribute used for the config?
By default, they should be the same, but it would be nice to do the following:
```
<div data-myattr
  data-config1={}
  data-config2={}
/>
```

and then in JS:
```
const initSearchApp = (rootElement) => {
    const { appId, ...config } = JSON.parse(
      rootElement.dataset[_camelCase("data-config1")]
    );
```

```
loadComponents(....., "data-myattr")
```

Reply:
https://github.com/inveniosoftware/invenio-search-ui/issues/125